### ISSUE TITLE
Examples with GGUI cannot be performed directly on Windows 10

### ISSUE THREAD

**Issue by Univstar** [taichi-dev/taichi#3023]
**Describe the bug**
After installing Taichi 0.8 on Windows 10 by 'pip install taichi', examples with GGUI cannot be performed.

**To Reproduce**

```shell
$ pip install taichi
$ ti example stable_fluid_ggui
```

**Log/Screenshots**

```
[Taichi] version 0.8.0, llvm 10.0.0, commit 181c9039, win, python 3.9.7

*******************************************
**      Taichi Programming Language      **
*******************************************

Docs:   https://taichi.rtfd.io/zh_CN/latest
GitHub: https://github.com/taichi-dev/taichi
Forum:  https://forum.taichi.graphics

Running example stable_fluid_ggui ...
[Taichi] Starting on arch=cuda
[E 09/27/21 15:42:41.717 8244] [embedded_device.cpp:taichi::lang::vulkan::EmbeddedVulkanDevice::create_instance@244] validation layers requested but not available


***********************************
* Taichi Compiler Stack Traceback *
***********************************
0x7fffc0be89ba: taichi::print_traceback in taichi_core.pyd
0x7fffc0abbc19: PyInit_taichi_core in taichi_core.pyd
0x7fffc0dd4a95: taichi::print_traceback in taichi_core.pyd
0x7fffc0dd1e35: taichi::print_traceback in taichi_core.pyd
0x7fffc0cf75f1: taichi::print_traceback in taichi_core.pyd
0x7fffc0cfe6a6: taichi::print_traceback in taichi_core.pyd
0x7fffc0d03067: taichi::print_traceback in taichi_core.pyd
0x7fffc0d02b38: taichi::print_traceback in taichi_core.pyd
0x7fffc098a92b: PyInit_taichi_core in taichi_core.pyd
0x7fffc0985562: PyInit_taichi_core in taichi_core.pyd
0x7fffc0983bfc: PyInit_taichi_core in taichi_core.pyd
0x7fffc098c9ef: PyInit_taichi_core in taichi_core.pyd
0x7fffc097e006: PyInit_taichi_core in taichi_core.pyd
0x7fffc09782bd: pybind11::error_already_set::discard_as_unraisable in taichi_core.pyd
0x7fffefeb59f2: PyArg_UnpackTuple in python39.dll
0x7fffefe91a2d: PyObject_MakeTpCall in python39.dll
0x7fffeff9d442: PyErr_FormatFromCauseTstate in python39.dll
0x7fffefe4d10f: Py_NewReference in python39.dll
0x7fffefebdabc: PyEval_EvalFrameDefault in python39.dll
0x7fffefe4fff0: PyFunction_Vectorcall in python39.dll
0x7fffefe4f037: PyFunction_Vectorcall in python39.dll
0x7fffeff2472d: PyList_Sort in python39.dll
0x7fffefe4c68d: PyType_GenericAlloc in python39.dll
0x7fffc097d6b6: PyInit_taichi_core in taichi_core.pyd
0x7fffefe4dcbd: Py_NewReference in python39.dll
0x7fffefebe049: PyEval_EvalFrameDefault in python39.dll
0x7fffefe4fff0: PyFunction_Vectorcall in python39.dll
0x7fffefe2cbcd: PyEval_EvalCodeWithName in python39.dll
0x7fffeff20b23: PyEval_EvalCodeEx in python39.dll
0x7fffeff20a81: PyEval_EvalCode in python39.dll
0x7fffeff20952: Py_c_prod in python39.dll
0x7fffeff2085b: Py_c_prod in python39.dll
0x7fffefe4d62d: Py_NewReference in python39.dll
0x7fffefebd6aa: PyEval_EvalFrameDefault in python39.dll
0x7fffefe4fff0: PyFunction_Vectorcall in python39.dll
0x7fffefe4d810: Py_NewReference in python39.dll
0x7fffefebd6aa: PyEval_EvalFrameDefault in python39.dll
0x7fffefe4fff0: PyFunction_Vectorcall in python39.dll
0x7fffefe4d810: Py_NewReference in python39.dll
0x7fffefebe049: PyEval_EvalFrameDefault in python39.dll
0x7fffefe4fff0: PyFunction_Vectorcall in python39.dll
0x7fffefe4d810: Py_NewReference in python39.dll
0x7fffefebe049: PyEval_EvalFrameDefault in python39.dll
0x7fffefe4fff0: PyFunction_Vectorcall in python39.dll
0x7fffefe4b3e4: PyType_GenericAlloc in python39.dll
0x7fffefe4d10f: Py_NewReference in python39.dll
0x7fffefebd6aa: PyEval_EvalFrameDefault in python39.dll
0x7fffefe4ef54: PyFunction_Vectorcall in python39.dll
0x7fffeff1fe2c: PyVectorcall_Call in python39.dll
0x7fffeff1fd1a: PyObject_Call in python39.dll
0x7fffefebe8c2: PyEval_EvalFrameDefault in python39.dll
0x7fffefe4fff0: PyFunction_Vectorcall in python39.dll
0x7fffefe4f037: PyFunction_Vectorcall in python39.dll
0x7fffefe2e85b: PyObject_FastCallDictTstate in python39.dll
0x7fffeff51b23: PyObject_Call_Prepend in python39.dll
0x7fffeff51a80: PyTime_GetSystemClock in python39.dll
0x7fffefe4dcbd: Py_NewReference in python39.dll
0x7fffefebd6aa: PyEval_EvalFrameDefault in python39.dll
0x7fffefe4d3ab: Py_NewReference in python39.dll
0x7fffefebd6aa: PyEval_EvalFrameDefault in python39.dll
0x7fffefe4fff0: PyFunction_Vectorcall in python39.dll
0x7fffefe2cbcd: PyEval_EvalCodeWithName in python39.dll
0x7fffeff20b23: PyEval_EvalCodeEx in python39.dll
0x7fffeff20a81: PyEval_EvalCode in python39.dll
0x7fffeff20952: Py_c_prod in python39.dll
0x7fffeff2085b: Py_c_prod in python39.dll
0x7fffefe4d62d: Py_NewReference in python39.dll
0x7fffefebd6aa: PyEval_EvalFrameDefault in python39.dll
0x7fffefe4fff0: PyFunction_Vectorcall in python39.dll
0x7fffefe4d810: Py_NewReference in python39.dll
0x7fffefebd6aa: PyEval_EvalFrameDefault in python39.dll
0x7fffefe4fff0: PyFunction_Vectorcall in python39.dll
0x7fffefe4f037: PyFunction_Vectorcall in python39.dll
0x7fffeff1fe2c: PyVectorcall_Call in python39.dll
0x7fffeff1fc27: PyObject_Call in python39.dll
0x7fffefe6c0c2: PyUnicode_EncodeFSDefault in python39.dll
0x7fffefe6cd9e: Py_RunMain in python39.dll
0x7fffefe6cca9: Py_RunMain in python39.dll
0x7fffefe6c6ce: Py_Main in python39.dll
0x7fffefe6c365: Py_Main in python39.dll
0x7ff61caf1268: Unknown Function in python.exe
0x7ff8484e7034: BaseThreadInitThunk in KERNEL32.DLL
0x7ff848762651: RtlUserThreadStart in ntdll.dll

Internal error occurred. Check out this page for possible solutions:
https://docs.taichi.graphics/docs/lang/articles/misc/install
[E 09/27/21 15:42:41.788 8244] Received signal 11 (SIGSEGV)


***********************************
* Taichi Compiler Stack Traceback *
***********************************
0x7fffc0be89ba: ?print_traceback@taichi@@YAXXZ in taichi_core.pyd
0x7fffc0abbc19: PyInit_taichi_core in taichi_core.pyd
0x7fffc0bd2cd8: PyInit_taichi_core in taichi_core.pyd
0x7ff8462a0ef5: _seh_filter_exe in ucrtbase.dll
0x7ff61caf2408: Unknown Function in python.exe
0x7ff83d8be390: __C_specific_handler in VCRUNTIME140.dll
0x7ff8487b20cf: __chkstk in ntdll.dll
0x7ff848761454: RtlRaiseException in ntdll.dll
0x7ff8487b0bfe: KiUserExceptionDispatcher in ntdll.dll

Internal error occurred. Check out this page for possible solutions:
https://docs.taichi.graphics/docs/lang/articles/misc/install
```

**Additional comments**
None.

**Comment by AmosChenZixuan** [comment]
Same problem on Ubuntu 20.04,  with Vulkan installed.
```text
VULKANINFO
==========

Vulkan Instance Version: 1.2.131


Instance Extensions: count = 18
====================
        VK_EXT_acquire_xlib_display            : extension revision 1
        VK_EXT_debug_report                    : extension revision 8
        VK_EXT_debug_utils                     : extension revision 2
        VK_EXT_direct_mode_display             : extension revision 1
        VK_EXT_display_surface_counter         : extension revision 1
        VK_KHR_device_group_creation           : extension revision 1
        VK_KHR_display                         : extension revision 23
        VK_KHR_external_fence_capabilities     : extension revision 1
        VK_KHR_external_memory_capabilities    : extension revision 1
        VK_KHR_external_semaphore_capabilities : extension revision 1
        VK_KHR_get_display_properties2         : extension revision 1
        VK_KHR_get_physical_device_properties2 : extension revision 1
        VK_KHR_get_surface_capabilities2       : extension revision 1
        VK_KHR_surface                         : extension revision 25
        VK_KHR_surface_protected_capabilities  : extension revision 1
        VK_KHR_wayland_surface                 : extension revision 6
        VK_KHR_xcb_surface                     : extension revision 6
        VK_KHR_xlib_surface                    : extension revision 6

Layers: count = 5
=======
VK_LAYER_LUNARG_standard_validation (LunarG Standard Validation Layer) Vulkan version 1.0.131, layer version 1:
        Layer Extensions: count = 0
        Devices: count = 2
                GPU id  : 0 (NVIDIA GeForce GTX 1060 6GB)
                Layer-Device Extensions: count = 0

```

**Comment by bobcao3** [comment]
Ah... Sorry we seem to kept debugging related layers as a depdency. We
should probably remove that in the next release, in the mean time,
installing the Vulkan SDK maybe a workaround.

**Comment by VickylastShao** [comment]
install Vulkan SDK for windows : https://vulkan.lunarg.com/sdk/home

**Comment by strongoier** [comment]
@Univstar We just released v0.8.1. Feel free to try out and report any further problems you meet.
